kombu/utils/compat.py

```diff
@@ -7,7 +7,11 @@
 from functools import wraps
 
 from contextlib import contextmanager
-import importlib_metadata
+
+try:
+    from importlib import metadata as importlib_metadata
```

Comment:
this import fails in python 3.7

Reply:
Correct.

Reply:
That's why it is in a try block.

Reply:
why not try import_metadata first?

Reply:
To prefer the standard library module if it is there, over the backport library.

Reply:
actually, kombu 4.6 ain't gonna get python 3.8 support. and 3.8 isn't stable yet.

Reply:
OK.